de-lib/src/main/java/org/iplantc/de/apps/client/presenter/sharing/AppSharingPresenter.java

```diff
@@ -222,14 +225,17 @@ private AppUnSharingRequestList buildUnSharingRequest() {
         FastMap<List<Sharing>> unSharingMap = permissionsPanel.getUnshareList();
 
         if (unSharingMap != null && unSharingMap.size() > 0) {
-            List<AppUnsharingRequest> requests = new ArrayList<>();
+            List<AppSharingRequest> requests = new ArrayList<>();
 
             for (String userName : unSharingMap.keySet()) {
                 List<Sharing> shareList = unSharingMap.get(userName);
 
-                AppUnsharingRequest unsharingRequest = appFactory.appUnSharingRequest().as();
-                unsharingRequest.setUser(userName);
-                unsharingRequest.setApps(buildUnshareAppPermissionList(shareList));
+                AppSharingRequest unsharingRequest = appFactory.appSharingRequest().as();
+                SharingSubject sharingSubject = shareFactory.getSharingSubject().as();
+                sharingSubject.setId(userName);
+                sharingSubject.setSourceId("ldap");
```

Comment:
Yeah, something like that.  I just wanted the smallest amount of code changes that will actually be used in this release since we're so close to the end. 😄